src/engine/renderer/tr_backend.cpp

```diff
@@ -5103,6 +5103,30 @@ const RenderCommand *Poly2dIndexedCommand::ExecuteSelf( ) const
 	return this + 1;
 }
 
+bool pushed = false;
+const RenderCommand *PushMatrixCommand::ExecuteSelf() const {
+	Tess_End();
+	if (!pushed) {
+		GL_PushMatrix();
+		pushed = true;
+	}
+	MatrixCopy( glState.projectionMatrix[ glState.stackIndex - 1 ],
+	            glState.projectionMatrix[ glState.stackIndex ] );
+	MatrixCopy( this->matrix,
```

Comment:
I think GL_LoadModelViewMatrix could be used

Reply:
Done.